src/trio/_tests/test_exports.py

```diff
@@ -117,6 +117,10 @@ def iter_modules(
 # won't be reflected in trio.socket, and this shouldn't cause downstream test
 # runs to start failing.
 @pytest.mark.redistributors_should_skip
+@pytest.mark.skipif(
```

Comment:
xfail so we can remove this once some tools support 3.14? (And so we don't have to redo this for 3.14b2 if nothings changed)

Reply:
I'm getting 14 XPASS locally if I do that (with 28 xfailed), so need to add some logic to distinguish them

Reply:
but probably less work to simply skipif all of 3.14.0b1 and then revisit later

Reply:
opened #3267 to track stuff